firefox-ios/Client/Assets/Images.xcassets/cookiesLarge.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "cookies-24.pdf",
```

Comment:
Looking at the file name it doesn't seem this image was not downloaded from the Acorn repo. Please ensure that all standard images are taken directly from the Acorn repo.

Reply:
took them from figma directly instead -_-